Android.mk

```diff
@@ -69,6 +69,13 @@ LOCAL_C_INCLUDES += $(LOCAL_PATH)/host/xtest \
 		$(LOCAL_PATH)/ta/aes_perf/include \
 		$(LOCAL_PATH)/ta/socket/include
 
+ifeq ($(CFG_SECURE_DATA_PATH),y)
+CFLAGS += -DWITH_SECURE_DATA_PATH
+
+srcs += sdp_basic.c
+LOCAL_C_INCLUDES += $(LOCAL_PATH)/ta/sdp_basic/include
```

Comment:
Add it always to `LOCAL_C_INCLUDES` for simplicity (same as `socket` which is also an optional feature of OP-TEE OS)